openapi/platform/platform.yaml

```diff
@@ -1245,6 +1245,22 @@ components:
                     description: The billing frequency
                 required:
                   - sku
+            status:
+              type: string
+              description: The status of the order
+              enum:
```

Comment:
For each of the options please provide a description telling the user when they would pick that status during a create/update, what happens as a result and what it means if we return this value to them.

Reply:
Also what is the default status that is used if one is not provided during create? Since it processes the order I assume it is approved.

Reply:
You actually can't provide the status during order create.

Reply:
The order is created in a state of ScheduledActivation with the scheduled activation date being determined by the corresponding time (currently hard coded within the backend).

Reply:
Lets document the field as being readonly with a default value of scheduledActivation for the time of creation. 

For filling in order forms I think we would have the user create the order as a draft and we would return a list of order forms to be filled out. They would then update the status when they are ready for it to be processed or sent for review. (Loose idea)